<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>706</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>807</width>
    <height>706</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>807</width>
    <height>706</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Coding</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Функции</string>
    </property>
    <widget class="QMenu" name="encodeMenu">
     <property name="title">
      <string>Кодирование</string>
     </property>
     <addaction name="GilberMoorEncode"/>
     <addaction name="HammingEncode"/>
    </widget>
    <widget class="QMenu" name="decodeMenu">
     <property name="title">
      <string>Декодирование</string>
     </property>
     <addaction name="GilberMoorDecode"/>
     <addaction name="HammingDecode"/>
    </widget>
    <addaction name="encodeMenu"/>
    <addaction name="decodeMenu"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="GilberMoorEncode">
   <property name="text">
    <string>Алгоритм Гильбера Мура</string>
   </property>
  </action>
  <action name="HammingEncode">
   <property name="text">
    <string>Алгоритм Хэмминга</string>
   </property>
  </action>
  <action name="GilberMoorDecode">
   <property name="text">
    <string>Алгоритм Гильбера Мура</string>
   </property>
  </action>
  <action name="HammingDecode">
   <property name="text">
    <string> Алгоритм Хэмминга</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
